### PR TITLE
added avatar component

### PR DESCRIPTION
**Pull request by anshikaa12**
>added avatar component
avatardoc

### CODE REVIEW

**Review by ArjunGTX — APPROVED** [review]
Looks good 👍🏻

File: components/avatars/avatar.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+.avatar-box{
+    border-radius:10%;
+    background-color: var(--primary-color);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+/************************/
+.badge-avatar{
```
Comment (prachi-sahani) [inline components/avatars/avatar.css]: can be named as "avatar-badge"(based on the naming convention you have used for other classes)

Comment (prachi-sahani) [inline components/avatars/avatar.css]: can be named as "avatar-badge"(based on the naming convention you have used for other classes)

File: components/avatars/avatar.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.online-badge{
```
Comment (prachi-sahani) [inline components/avatars/avatar.css]: same as mentioned above - badge-online(as you are varying the type of badges) or you can even name it as "avatar-badge-online"(this name seems a little big, so not sure if that will be fine 😄 )

**Review by prachi-sahani — COMMENTED** [review]
Just a minor feedback - check for compliance of background color of badge and the text color(acc. to me the text is not so clear)

Except the points mentioned, this component is looking good. Well done! 👍🏻

File: components/avatars/avatar.css (hunk 1 above)
Comment (anshikaa12) [reply components/avatars/avatar.css]: sure will do this!

File: components/avatars/avatar.css (hunk 2 above)
Comment (anshikaa12) [reply components/avatars/avatar.css]: did all the changes! thanks!

File: components/avatars/avatar.css (hunk 1 above)
Comment (anshikaa12) [reply components/avatars/avatar.css]: did it!

**Comment by Bharatrathod-81** [comment]
Looks good

**Review by Bharatrathod-81 — COMMENTED** [review]
Great work

**Pull request merged**